src/main/java/org/folio/dcb/service/CirculationRequestService.java

```diff
@@ -0,0 +1,8 @@
+package org.folio.dcb.service;
+
+import org.folio.dcb.domain.dto.CirculationRequest;
+
+public interface CirculationRequestService {
+  CirculationRequest fetchRequestById(String requestId);
```

Comment:
I see fetchRequestById is not being called anywhere outside the class. Do you think we need to have it here